tests/integration/devfile/cmd_devfile_catalog_test.go

```diff
@@ -103,9 +102,7 @@ var _ = Describe("odo devfile catalog command tests", func() {
 	Context("When executing catalog describe component with a component name with multiple components", func() {
 		It("should print multiple devfiles from different registries", func() {
 			helper.CmdShouldPass("odo", "registry", "add", registryName, addRegistryURL)
-			output := helper.CmdShouldPass("odo", "registry", "list")
```

Comment:
We have a separate test file for devfile registry and checking registry commands in catalog file just duplicating the steps. As we are adding the same registry url `https://github.com/odo-devfiles/registry` in [devfile-regsitry](https://github.com/openshift/odo/blob/master/tests/integration/devfile/cmd_devfile_registry_test.go#L61-L63) So removing it from here.